packages/block-editor/src/components/block-lock/modal.js

```diff
@@ -100,7 +100,7 @@ export default function BlockLockModal( { clientId, onClose } ) {
 				<fieldset className="block-editor-block-lock-modal__options">
 					<legend>
 						{ __(
-							'Choose specific attributes to restrict or lock all available options.'
+							'Choose "Lock all" to disable all actions, or select specific features to lock.'
```

Comment:
It's best to make this `legend` text shorter as suggested in the comments.

Reply:
Yes I agree!
> `Choose the features to lock.` sounds good too. Alternatively it can also be `Select the features you want to lock.`

Here, I had suggested one more alternative which we haven't discussed yet.
Hence, I did not shorten it further since the choice of message was pending